101-create-backup-vault-policy/azuredeploy.json

```diff
@@ -0,0 +1,180 @@
+        },
+        "location": {
+            "type": "string",
+            "allowedValues": [
+                "East US",
+                "East US 2",
+                "West US",
+                "Central US",
+                "North Central US",
+                "South Central US",
+                "North Europe",
+                "West Europe",
+                "East Asia",
+                "South East Asia",
+                "East China",
+                "West Japan",
+                "Australia East",
+                "Australia Southeast",
+                "Brazil South",
+                "Japan East",
+                "Japan West"
+            ],
+            "metadata": {
+                "description": "This is the location where the backup vault will be created"
+            }
+        },
+        "skuName": {
+            "type": "string",
+            "defaultValue": "Standard",
+            "allowedValues": [
+                "Standard",
+                "Premium"
+            ],
+            "metadata": {
+                "description": "SKU for the vault"
+            }
+        },
+        "policyName": {
+            "type": "string",
+            "metadata": {
+                "description": "Name of the Backup Policy"
+            }
+        },
+        "workloadType": {
+            "type": "string",
+            "allowedValues": [
+                "IaasVM"
+            ],
+            "metadata": {
+                "description": "Type of Workload. Currently Supported only IaasVM."
+            }
+        },
+        "backupType": {
+            "type": "string",
+            "allowedValues": [
+                "Full"
+            ],
+            "metadata": {
+                "description": "Type of Backup. Full Backup is supported."
+            }
+        },
+        "scheduleRun": {
+            "type": "string",
+            "allowedValues": [
+                "Daily",
+                "Weekly"
+            ],
+            "metadata": {
+                "description": "Backup frequency specifies the schedule at which a backup job is triggered and a recovery point is created."
+            }
+        },
+        "scheduleRunDays": {
+            "type": "array",
+            "metadata": {
+                "description": "Backup Schedule will run on array of Days like, Monday, Tuesday etc. Applies in Weekly Backup Type only."
+            }
+        },
+        "scheduleRunTimes": {
+            "type": "array",
+            "metadata": {
+                "description": "Times in day when backup should be triggered. e.g. 01:00, 13:00. This will be used in LTR too for daily, weekly, monthly and yearly backup."
+            }
+        },
+        "dailyRetentionDurationCount": {
+            "type": "int",
+            "metadata": {
+                "description": "Number of days you want to retain the backup"
+            }
+        },
+        "dailyRetentionDurationType": {
+            "type": "string",
```

Comment:
This could become a variable since there is only one possible value here and user input doesn't really change anything